h3/src/quic.rs

```diff
@@ -103,4 +106,7 @@ pub trait BidiStream<B: Buf>: SendStream<B> + RecvStream {
 
     /// Split this stream into two halves.
     fn split(self) -> (Self::SendStream, Self::RecvStream);
+
+    /// Get QUIC send stream id
+    fn id(&self) -> u64;
```

Comment:
This is already being added to `SendStream` above, so it shouldn't be needed here, I don't think.

Reply:
Yeah, plus this isn't needed as long as we don't have to track Dynamic table references in headers. I'll get rid of those for now.